interface/modules/zend_modules/module/Patientvalidation/Module.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * User: Dror Golan
+ * Date: 03/07/16
+ * Time: 10:42
+ */
```

Comment:
Need a standard header. See here for an example:
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#Page-Level_DocBlock